framework/src/base/MooseApp.C

```diff
@@ -215,6 +221,19 @@ validParams<MooseApp>()
       false,
       "Keep standard output from all processors when running in parallel");
 
+  // Options for debugging
+  params.addCommandLineParam<std::string>("start_in_debugger",
+                                          "--start-in-debugger [debugger]",
```

Comment:
Should be \<debugger\>, brackets mean optional argument.

Reply:
👍